dev/rebuild_hbase.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# Downloads the specified HBase version sources into PWD, extracts it, 
+# then rebuilds and installs the artifacts with -Dhadoop.profile=3.0
+
+if [ $# -ne 1 ]
+  then
+    echo "Supply the Hbase version as paramater i.e.: rebuild_hbase.sh 2.2.6 "
+fi
+
+wget https://downloads.apache.org/hbase/$1/hbase-$1-src.tar.gz
```

Comment:
Infra requests that downloads.a.o/archive.a.o is only used for downloading KEYS, checksums, and signatures.

Can you adapt the solution I have in https://github.com/apache/phoenix-queryserver/blob/master/python-phoenixdb/dev-support/cache-apache-project-artifact.sh to this script as well, please?

Reply:
Sure, I will do that before commit.